test/unittest/StatisticsBackend/BackendDumpTests.cpp

```diff
@@ -19,33 +19,11 @@
 
 #include <StatisticsBackend.hpp>
 #include <StatisticsBackendData.hpp>
+#include <DatabaseUtils.hpp>
 
 using namespace eprosima::statistics_backend;
 using namespace eprosima::statistics_backend::database;
 
-void load_file(
-        std::string filename,
-        DatabaseDump& dump)
-{
-    constexpr const char* DESCRIPTION_TAG = "description";
-
-    // Check if the file exists
-    std::ifstream file(filename);
-    if (!file.good())
-    {
-        FAIL() << "File " + filename + " does not exist";
-    }
-
-    // Get the json
-    file >> dump;
-
-    // Erase the description tag if existing
-    if (dump.contains(DESCRIPTION_TAG))
-    {
-        dump.erase(DESCRIPTION_TAG);
-    }
-}
-
 TEST(backend_dump_tests, reset)
 {
     constexpr const char* DUMP_FILE = "../Resources/simple_dump.json";
```

Comment:
Use here the file declared in `DatabaseUtils`.
Apply the same change on all the test on this file.